e2e/certification.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { test, expect } from '@playwright/test';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/certification/certifieduser/responsive-web-design');
+});
+
+test.describe('Certification page', () => {
+  test('The certificate page has a donation section', async ({ page }) => {
+    const donation = page.getByTestId('donation-section');
+    await expect(donation).toBeVisible();
+
+    const donationText = page.getByTestId('donation-text');
+    await expect(donationText).toHaveText(
+      `Only you can see this message. Congratulations on earning this certification. It's no easy task. Running freeCodeCamp isn't easy either. Nor is it cheap. Help us help you and many other people around the world. Make a tax-deductible supporting donation to our charity today.`
+    );
```

Comment:
It's better if you get the text from the translations: `client/i18n/locales/english/translations.json`

Reply:
Fixed